package.json

```diff
@@ -28,6 +28,66 @@
     "*"
   ],
   "contributes": {
+    "commands": [
+      {
+        "command": "emojisense.quickEmoji",
+        "title": "Pick an emoji and insert in editor",
+        "category": "Emojisense"
+      },
+      {
+        "command": "emojisense.quickEmojitext",
+        "title": "Pick an :emoji: and insert in editor",
+        "category": "Emojisense"
+      },
+      {
+        "command": "emojisense.quickEmojiTerminal",
+        "title": "Pick an emoji and insert in terminal",
+        "category": "Emojisense"
+      },
+      {
+        "command": "emojisense.quickEmojitextTerminal",
+        "title": "Pick an :emoji: and insert in terminal",
+        "category": "Emojisense"
+      }
+    ],
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "emojisense.quickEmojiTerminal",
+          "when": "panelFocus && activeViewlet == 'workbench.panel.terminal'"
+        },
+        {
+          "command": "emojisense.quickEmojitextTerminal",
+          "when": "panelFocus && activeViewlet == 'workbench.panel.terminal'"
+        }
+      ]
+    },
+    "keybindings": [
+      {
+        "mac": "cmd+[Period]",
```

Comment:
This keybinding is already used to trigger code actions. Any ideas for a different one that does not conflict with VS Code's built-in keybindings?

Reply:
Didnt know about that one! :)

Reply:
I tried out <kbd>cmd+[Minus]</kbd> on my computer, what do you think about that one?
(this is  <kbd>cmd+ )</kbd> on my french keyboard)